packages/satori-js/client.ts

```diff
@@ -125,14 +125,99 @@ export class Client {
     return this.apiClient.satoriGetExperiments(session.token, names);
   }
 
+  /** Get a single flag for this identity. */
+  async getFlag(session: Session, name: string, defaultValue?: string) {
```

Comment:
One other thing -- we'll want an overload (or a differently named function since it's TS) that doesn't accept a default value and just throws an exception if the flag doesn't exist.

Probably 80% of the time the default value one will be used, but there may be cases where users want to actually know if the flag didn't exist on the server vs. had a null value.